cluster/config-defaults.yaml

```diff
@@ -62,11 +62,22 @@ skipper_redis_memory: "100Mi"
 
 # skipper api GW features
 enable_apimonitoring: "true"                       # TODO(sszuecs): cleanup candidate to reduce amount of branches in deployment
+
+# skipper east-west feature
+# enable_skipper_eastwest is the legacy feature gate for the automatic
+# ingress.cluster.local addresses created by skipper.
+# enable_skipper_eastwest_dns only enables DNS and assumes users define the
+# ingress.cluster.local names explicitly on ingress/routegroup/stacksets
 {{if eq .Environment "production"}}
 enable_skipper_eastwest: "false"
+enable_skipper_eastwest_dns: "false"
```

Comment:
We need to make sure that we set this in current clusters via CR :)

Reply:
This value has been set for all clusters which currently has `enable_skipper_eastwest: true`